client/extensions/woocommerce/app/products/product-form-details-card.js

```diff
@@ -65,9 +66,10 @@ export default class ProductFormDetailsCard extends Component {
 					</FormLabel>
 				</div>
 				<div className="products__product-form-details-wrapper">
-					<div className="products__product-form-details-images">
-
-					</div>
+					<ProductFormImages
+						product={ product }
```

Comment:
It seems ProductFormImages really on needs the product 1) for its images property and 2) to pass it into editProduct.  Instead of passing the entire product, for better decoupling, please consider passing instead 1) the images property and 2) a new member function (that lives in this component) that passes the product and images into editProduct for ProductFormImages